{"UniqueId":"2284b46b-b8db-49b0-8c89-737726a989c9","AppType":1,"DateTime":"4/3/2020 12:14:19 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"0628b9e6-9a38-4dbc-807a-0951eb20504b","AppType":1,"DateTime":"4/3/2020 12:14:19 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"61b247ed-2d00-42df-a432-9b9fcebc139c","AppType":1,"DateTime":"4/3/2020 12:14:35 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"9f058058-b931-4449-93d7-f7191d882182","AppType":1,"DateTime":"4/3/2020 12:14:35 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"84ab9237-a81b-4551-b177-363fe827a40f","AppType":1,"DateTime":"4/3/2020 12:14:50 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"bc0594fe-03be-4871-acc8-85a2e1bda621","AppType":1,"DateTime":"4/3/2020 12:14:50 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"46a9267f-df25-4b52-8463-99585138fce2","AppType":1,"DateTime":"4/3/2020 12:14:51 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"81a7d589-21ac-49b0-9d33-b97adb79867a","AppType":1,"DateTime":"4/3/2020 12:14:51 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"0f0b7045-aa49-474a-8120-25878570cc1c","AppType":1,"DateTime":"4/3/2020 12:14:51 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"5f3ee2ff-0350-489a-a857-f2dd7009f976","AppType":1,"DateTime":"4/3/2020 12:14:51 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"8132e26e-1644-4fd1-94b0-b7ac12161b7f","AppType":1,"DateTime":"4/3/2020 12:15:01 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"1e87f4d1-f315-4510-9979-74dae9fc9e89","AppType":1,"DateTime":"4/3/2020 12:15:01 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"a807b52c-f91a-4687-bb3c-e6bf618c1c5b","AppType":1,"DateTime":"4/3/2020 12:15:11 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
{"UniqueId":"f71f5c61-0555-4009-abf1-08fbe9b64fb1","AppType":1,"DateTime":"4/3/2020 12:15:11 PM","Namespace":"ReshimgathiServices.Controllers.LoginController","Controller":"Login","IsExceptionHandled":null,"Exception":"Object reference not set to an instance of an object.","ErrorCode":"-2147467261","FileName":"C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs","MethodName":"MoveNext","LineNo":41,"ColumnNo":13,"StackTrace":"   at ReshimgathiServices.RAuthFilter.<AuthenticateAsync>d__2.MoveNext() in C:\\E\\VS-Programs\\Organizations\\Reshimgathi\\ReshimgathiServices\\ReshimgathiServices\\ReshimgathiServices\\RAuthFilter.cs:line 41\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","Host":"localhost","Version":"1.1"}
